pages/dkp/kaptain/2.1.0/install/air-gapped-2.2/index.md

```diff
@@ -104,7 +104,7 @@ If you added Kaptain after installing DKP, you must make it available by rerunni
     mkdir kaptain-catalog-applications
     tar -xvf kaptain-catalog-applications.tar.gz --strip-components 1 -C kaptain-catalog-applications
     ```
-
+<!-- Do we need to update anything on the file location above? update/decompress the link?-->
```

Comment:
This is the catalog and doesn't contain Minio references. No changes are needed.